### PR TITLE
Update ex4-apache.md

### PR DESCRIPTION
**Pull request by Petraea**
Make a2enmod instruction slightly clearer.

### CODE REVIEW

**Comment by bgruening** [comment]
Merged the other one :)

**Comment by nsoranzo** [comment]
@bgruening Not sure why you closed this, seems still valid to me.

File: sessions/03-production-basics/ex4-apache.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ To make Apache serve as a reverse proxy, we will need to enable the modules:
 - `mod_proxy`
 - `mod_proxy_http`
 
-These are enabled using the `a2enmod` command:
+These are enabled using the `a2enmod` command (without the mod_ prefix):
```
Comment (nsoranzo) [inline sessions/03-production-basics/ex4-apache.md]: I'd add \` quotes around `mod_`.

**Comment by nsoranzo** [comment]
Thanks @Petraea.

**Pull request merged**